website/README.md

```diff
@@ -118,6 +118,44 @@ The code for the client is located under `src/client/`. The webpack build create
 
 Scripts related to deployment, like the Dockerfile, are under `deploy`. Note: even though the scripts are under `deploy`, you must run them from the **root of the repository**; they are sensitive to `cwd`.
 
+## Query Parameters
+
+You can pass certain parameters in the Snack URL to customize the behavior of the Snack.
+
+Here is a summary of all the parameter options that you can use:
+
+> All query parameter values must be URL encoded
+
+| parameter | default | description | type |
+| -------- | ------- | ----------- | ------ |
+| **dependencies** | | Comma separated list of <name>@<version>. | [SnackDependency](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L64) |
```

Comment:
I think we need to escape the `<` and `>` by adding a backslash to it, e.g. `\<`. Right now, github filters it because it thinks its a HTML tag

<img width="1014" alt="Screenshot 2022-05-27 at 15 51 06" src="https://user-images.githubusercontent.com/1203991/170712897-9311340e-9fe2-4f4a-95a2-2037ea3605d6.png">

Reply:
good catch